<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportItemWidget</class>
 <widget class="QWidget" name="ReportItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="event_lb">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="status_lb">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>9</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_lb">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>9</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
